test/storage_test.exs

```diff
@@ -8,12 +8,14 @@ defmodule Neo4Ecto.StorageTest do
   end
 
   describe "storage_up/1" do
+    @tag :skip
```

Comment:
I think we can remove those `:skips` now since there won't be tests in parallel 🤔

Reply:
It's still failing because we don't have Neo4j desktop.

Reply:
Oh forgot about that. :shipit: